packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -389,4 +444,133 @@ describe('backupNg', () => {
     })
     expect(vmTask.data.id).toBe(vmId)
   })
+
+  test('execute three times a delta backup with 2 remotes, 2 as retention and 2 as fullInterval', async () => {
+    jest.setTimeout(6e4)
+    const vmId = config.vms.vmToBackup
+    if (xo.objects.all[vmId] === undefined) throw new Error('no vm to backup')
```

Comment:
No need to check if it exists. The issue is if it's `undefined` all VMs will be backed up .  I suggest:

```js
    const {
      vms: { default: defaultVm, vmToBackup = defaultVm },
      remotes: { default: defaultRemote, remote1, remote2 = defaultRemote }
    } = config
    expect(vmToBackup).not.toBe(undefined)
    expect(remote1).not.toBe(undefined)
    expect(remote2).not.toBe(undefined)
```